README.md

```diff
@@ -22,7 +22,11 @@ Or install it yourself as:
 
 ```ruby
 
-summary_document = TelephoneAppointments::SummaryDocumentActivity.new(appointment_id: id, owner_uid: uid)
+summary_document = TelephoneAppointments::SummaryDocumentActivity.new(
+  appointment_id: id,
+  owner_uid: uid, 
+  message: message # 'postal' or 'digital'
```

Comment:
Not sure that `message` is the right terminology for this. How about `type`? Or maybe even `postal: true|false` and perhaps defaulting to a sane value?